lib/tools/apk-utils.js

```diff
@@ -474,12 +474,81 @@ apkUtilsMethods.getDeviceLocale = async function () {
 
 /**
  * Set the locale name of the device under test.
+ * Compatibility for appium-android-driver 1.32.4-
```

Comment:
please do not add explicit driver versions, since not only android driver depends on this module